docs/manual/common/concepts/MicroserviceSystemDesign.md

```diff
@@ -0,0 +1,7 @@
+# Designing your microservices system
+
+Bonér's [Reactive Microservices Architecture: Design Principles for Distributed Systems](http://www.oreilly.com/programming/free/reactive-microservices-architecture.html) is fundamental reading for microservices systems architects. It takes time and experience to become skilled in this complex domain. Lagom provides practical guidance by offering opinionated APIs, implementations of supporting features, and appropriate defaults.
+
+If you are introducing microservices into an existing monolith, you might need to vary your approach based on its architecture. For example, if it is reasonably well-designed, you might be able to first de-couple its components and focus on migrating functionality a bit at a time. Whether that is the case or not, we recommend starting small. First, identify a need for a simple microservice that can consume asynchronous messages. It needn't be complex or even provide a lot of value. The simplicity reduces the risk associated with deployment and can provide a quick win. Then, at the architectural level pull out a core service that can be compartmentalized, and decide how to divide it into microservices. When you attack the problem a piece at a time, you and your team will learn as you go and will become increasingly effective.
```

Comment:
"If you are introducing microservices into an existing monolith" <-- this sentence sounds weird, you are probably extracting microservices from a monolith. Or introducing microservices arechitecture into a tema/organization that is currently dealing with a monolith.